tests/test_writer.py

```diff
@@ -0,0 +1,59 @@
+import os
+import fcntl
+from multiprocessing import Process, Queue
+from datetime import datetime
+import pytest
+from warcprox.mitmproxy import ProxyingRecorder
+from warcprox.warcproxy import RecordedUrl
+from warcprox.writer import WarcWriter
+from warcprox import Options
+
+recorder = ProxyingRecorder(None, None, 'sha1', url='http://example.com')
+
+recorded_url = RecordedUrl(url='http://example.com', content_type='text/plain',
```

Comment:
For testing it'd be better to use ip addresses in a private (or loopback) range rather than ones that might point somewhere out there on the internet. Maybe 127.0.0.2 and 127.0.0.3 or something.

Reply:
Good point